src/panes.ts

```diff
@@ -142,6 +142,8 @@ export type FieldsPane = Pane<
       title: string
       provider?: ProviderMetadata
     }
```

Comment:
maybe add a `description` field to the header?

Reply:
https://github.com/seamapi/seamapi-types/pull/73/commits/dc8a0c04f138987b0cf0c6630ee6dac14d9efbcf